lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.py

```diff
@@ -0,0 +1,26 @@
+import maya.api.OpenMaya as om
+from pxr import Sdf
+import mayaUsd
+import ufe
+
+# Adapted from testMayaUsdSchemasMayaReference.py.
+def createMayaReferencePrim(ufePathStr, mayaReferencePath, mayaNamespace):
+    '''Create a Maya reference prim parented to the argument path.'''
+
+    # Append reference name to parent Sdf path.
+    ufePath = ufe.PathString.path(ufePathStr)
+    parentPath = ufePath.segments[1]
+    primPath = Sdf.AssetPath(str(parentPath)+'/MayaReference')
+
+    stage = mayaUsd.ufe.getStage(ufePathStr)
+    prim = stage.DefinePrim(primPath.path, 'MayaReference')
+    if not prim.IsValid():
+        om.MGlobal.displayError("Could not create MayaReference prim under %s" % ufePathStr)
+        return
+
+    mayaReferenceAttr = prim.CreateAttribute('mayaReference', Sdf.ValueTypeNames.Asset)
+    mayaReferenceAttr.Set(mayaReferencePath)
```

Comment:
We got a MayaReference path, a MayaReference custom prim type and mayaReference attribute. It's a bit confusing :) I suppose it cannot be avoided.

Reply:
Those names were chosen by Animal Logic, I won't change them.
https://github.com/Autodesk/maya-usd/blob/600c31abcd68ef3fe0047b28fb5f33f6edd0872f/lib/usd/schemas/schema.usda.in#L22